user-guide.rst

```diff
@@ -4,24 +4,28 @@ User Guide & FAQ
 Frequently Asked Questions
 --------------------------
 
-.. rubric:: Why are there different Scenario Explorer for specific projects?
+.. rubric:: Why are there different *Scenario Explorer* instances for specific projects?
 
 Each project usually brings together different consortium partners and uses a
 project-specific list of variable and regions (see :ref:`project-templates`).
 
 It is therefore easier to keep projects separate, with its own database **platform**,
 *Scenario Explorer* instance and related GitHub repository (see :ref:`scenario-processing`).
 
-.. rubric:: What is the level of spatial granularity?
+.. rubric:: What is the level of spatial granularity of scenarios?
 
```

Comment:
I'd start with something along the lines of:
The Scenario Explorer as a tool is agnostic to the spatial resolution of the data.

Reply:
Following your advice to make the text less technical (instead of adding the word "agnostic").